test/integration/puppetlabs/services/jruby/class_info_test.clj

```diff
@@ -167,8 +163,7 @@
                                                [env-2-dir-and-manifests
                                                 env-1-mod-1-dir-and-manifests
                                                 env-1-mod-2-dir-and-manifests])}]
-              (testing "all environments"
-                (is (= expected-envs-info (get-class-info))))
+              (puppet-env/mark-all-environments-expired! env-registry)
```

Comment:
A few of these calls were added to these tests in order to compensate for the `Puppet.settings.clear_environment_settings` call being removed from master.rb's `getClassInfoForEnvironment` method.  See the comment in that method for more context.